src/il_representations/scripts/il_train.py

```diff
@@ -66,7 +66,6 @@ def bc_defaults():
     # regularisation
     ent_weight = 1e-3
     l2_weight = 1e-5
-    n_trajs = None
```

Comment:
Are you deleting `n_trajs` because we would not further ablate on this in the future? I think it makes sense, just want to confirm.

Reply:
In d598017cd656ebc9558bf05bbf9620ffc6a2eb9d I changed the `n_trajs` enforcement mechanism. Instead of having a separate `n_trajs` option for each algorithm, we now have a single `env_data.wds_n_trajs` option which applies per-dataset. This way you can enforce a separate `n_trajs` value for each dataset, if training with multiple datasets.

Reply:
Possibly this should be mentioned somewhere? I also added a few namedconfigs that show how to use the new mechanism, so I hope it will be somewhat self-explanatory. It looks like this change still broke `joint_training.py`, though, so I'll have to fix that!